harness/determined/experimental/_native.py

```diff
@@ -67,16 +55,16 @@ def set_command_default(
     return command
 
 
-def create_experiment(
+def _create_experiment(
     config: Optional[Dict[str, Any]],
     context_dir: str,
     command: Optional[List[str]],
-    test_mode: bool = False,
+    test: bool = False,
     master_url: Optional[str] = None,
 ) -> Optional[int]:
     """Submit an experiment to the Determined master.
 
-    Alternatively, use det.create() with a mode argument of "submit".
+    Users should call det.create() with ``submit=True``.
```

Comment:
Perhaps just remove this docstring since it will be internal only.

Reply:
done